src/qt/locale/bitcoin_it.ts

```diff
@@ -2622,71 +2622,71 @@ p, li { white-space: pre-wrap; }
-        <translation type="unfinished"></translation>
+        <translation>alta</translation>
     </message>
     <message>
         <source>Are you sure you want to send?</source>
-        <translation type="unfinished"></translation>
+        <translation>Sei sicuro di voler inviare?</translation>
     </message>
     <message>
         <source>Total Amount %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Importo Totale %1</translation>
     </message>
     <message>
         <source>Confirm send coins</source>
-        <translation type="unfinished"></translation>
+        <translation>Conferma invio</translation>
     </message>
     <message>
         <source>The recipient address is not valid. Please recheck.</source>
-        <translation type="unfinished"></translation>
+        <translation>L&apos;indirizzo del destinatario non é valido, controlla di nuovo.</translation>
     </message>
     <message>
         <source>The amount to pay must be larger than 0.</source>
-        <translation type="unfinished"></translation>
+        <translation>L&apos;importo da inviare deve essere superiore a 0.</translation>
     </message>
     <message>
         <source>The total exceeds your balance when the %1 transaction fee is included.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il totale è superiore al saldo disponibile quando le commissioni di transazione di %1 sono incluse.</translation>
     </message>
     <message>
         <source>Duplicate address found: addresses should only be used once each.</source>
-        <translation type="unfinished"></translation>
+        <translation>Indirizzo già inserito: lo stesso indirizzo può essere inserito solo una volta.</translation>
     </message>
     <message>
         <source>A fee higher than %1 is considered an absurdly high fee.</source>
-        <translation type="unfinished"></translation>
+        <translation>Commissioni di transazione superiori a %1 sono irragionevolmente alte.</translation>
     </message>
     <message>
         <source>Pay only the required fee of %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Paga solo le commissioni di transazione richieste di %1</translation>
     </message>
     <message>
         <source>Confirm custom change address</source>
-        <translation type="unfinished"></translation>
+        <translation>Conferma l&apos;indirizzo di scambio</translation>
     </message>
     <message>
         <source>The address you selected for change is not part of this wallet. Any or all funds in your wallet may be sent to this address. Are you sure?</source>
-        <translation type="unfinished"></translation>
+        <translation>L&apos;indirizzo che hai selezionato come indirizzo di scambio non é parte di questo portafoglio. Parte o tutti i tuoi fondi potrebbero essere inviati a questo indirizzo. Sei sicuro?</translation>
```

Comment:
<translation>L&apos;indirizzo che hai selezionato come indirizzo di scambio non appartiene a questo portafoglio. I  tuoi fondi potrebbero essere in parte o totalmente inviati a questo indirizzo. Sei sicuro?</translation>